src/components/Login.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import {User} from 'darity-state';
+import 'whatwg-fetch';
+
+class Login extends Component {
+  state = {
+            username: '', 
+            password: ''
+          }
+
+  usernameChange(e){
```

Comment:
Change these to fat arrow functions so that you don't have to use bind in the render method. I.E.

usernameChange = e => this.setState({username: e.target.value})

Then you can change line 42 with onChange={this.usernameChange}